synapse/models/inet.py

```diff
@@ -255,6 +256,25 @@ def getDataModel():
 
             ]),
 
+            ('inet:netfile',{},[
```

Comment:
Possibly.  If so, lets noodle it and do that in another delta ( adding a new secondary prop is pretty harmless )